packages/address-book-controller/package.json

```diff
@@ -22,9 +22,10 @@
   ],
   "scripts": {
     "changelog:validate": "../../scripts/validate-changelog.sh @metamask/address-book-controller",
-    "doc": "../../scripts/doc.sh",
+    "doc": "typedoc",
```

Comment:
Can we name this `build:docs` to stay consistent with the module template?

Reply:
Oops, I hadn't noticed. Renamed in [3bf6120](https://github.com/MetaMask/controllers/pull/831/commits/3bf612091194a0b42807a87ce8fac2e038b7d171).

Reply:
Nit: It looks like only the top-level script has been renamed, not the package scripts

Reply:
Good call! Changed those as well here: [0b957f8](https://github.com/MetaMask/controllers/pull/831/commits/0b957f89978edfa7b40483a557526a19ffca79a0)